packages/checkout-ui-extensions/src/components/TextBlock/README.md

```diff
@@ -8,7 +8,7 @@ optional = ?
 | Name | Type | Description |
 | --- | --- | --- |
 | size? | <code>"extraSmall" &#124; "small" &#124; "base" &#124; "large" &#124; "extraLarge" &#124; "medium"</code> | Size of the text |
-| emphasis? | <code>"stress" &#124; "offset" &#124; "strong"</code> | Use to emphasize a word or a group of words compared to other nearby text.<br /><br /><code>"stress"</code>: Indicate emphatic stress. By default the text will be italic and render a `<em>` tag in an HTML host.<br /><br /><code>"offset"</code>: Indicate an offset from the normal prose of the text. Typically used to indicate a foreign word, fictional character thoughts, or when the text refers to the definition of a word instead of representing its semantic meaning. By default the text will be italic and render a `<i>` tag in an HTML host.<br /><br /><code>"strong"</code>: Indicate strong importance, seriousness, or urgency. By default the text will be bold and render a `<strong>` tag in an HTML host. |
+| emphasis? | <code>"italic" &#124; "bold"</code> | Use to emphasize a word or a group of words compared to other nearby text.<br /><br /><code>"italic"</code>: Set the text in italic. Combine with an accessibilyRole of 'offset' or 'stress' to add more meaning to the text.<br /><br /><code>"bold"</code>: Set the text in bold. |
```

Comment:
```suggestion
| emphasis? | <code>"italic" &#124; "bold"</code> | Use to emphasize a word or a group of words compared to other nearby text.<br /><br /><code>"italic"</code>: Set the text in italic.<br /><br /><code>"bold"</code>: Set the text in bold. |
```

Reply:
@jffortier the `Emphasis` is a shared type so the descriptions for both `TextBlock` and `Text` would need to be same no?

Reply:
Ya, the `TextBlockProps` don't include the `accessibilityRole` so you might need to adjust the description for `italic` and `bold` to not mention `accessibilityRole`.

Reply:
Yep, it gets pulled into `shopify.dev` as well. If you open a PR in `checkout-web` with the ts-doc changes I can bring that update in and regen the docs with it.